templates/geosafe/analysis/create.html

```diff
@@ -976,6 +979,45 @@
             });
         }
 
+        function set_transparency() {
+            var exposure_id = $(".exposure.section a.selected").attr('data-id');
+            var hazard_id = $(".hazard.section a.selected").attr('data-id');
+            //make sure the exposure or hazard layers has selected.
+            if (exposure_id == undefined && hazard_id == undefined)
+            {
+                return;
+            }
+            exposure_layer = add_layer(map, layer_tiles_url, exposure_layer);
+            hazard_layer = add_layer(map, layer_tiles_url, hazard_layer);
+            $('#transparency-option').popover({
+                    html: true,
+                    placement: 'right',
+                    title: 'Set Transparency',
+                    content: function () {
+                        return $('#popover-transparency').html();
+                    }
+                }).click(function () {
+                    //Using slider from https://github.com/seiyria/bootstrap-slider
+                    //there are two slider: jquery and seiyria's version
+                    //calling slider with bootstrapSlider to distinguish between them according to: http://stackoverflow.com/questions/26691018/using-bootstrap-slider-with-jquery-ui
+                    $('#exposure-slider').bootstrapSlider().on('slide', function (ev) {
+                        slider_val = ev.value;
+                        exposure_layer.setOpacity(slider_val/10);
+                    });
+                    if (slider_val) {
+                        $('#exposure-slider').bootstrapSlider('setValue', slider_val);
+                    }
+                    /*
+                    $('#hazard-slider').bootstrapSlider().on('slide', function (ev) {
```

Comment:
@lucernae it is reserved for #71 
i'll continue work on it and remove the comment on the next PR.
So, i'll close the ticket